app/assets/stylesheets/characters.scss

```diff
@@ -1,3 +1,13 @@
+// Place all the styles related to the characters controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+// A94D2C
+
+.flex{
```

Comment:
This class name is to attached to a particular css property, please update accordingly